GVRf/Framework/src/org/gearvrf/GVRContext.java

```diff
@@ -517,7 +518,16 @@ public GVRSceneObject loadJassimpModel(String assetFile) throws IOException {
     }
 
     public GVRSceneObject loadJassimpModel(String assetFile, EnumSet<GVRImportSettings> settings) throws IOException {
-        return GVRImporter.loadJassimpModel(this, assetFile, settings);
+        return GVRImporter.loadJassimpModel(this, assetFile, GVRResourceVolume.VolumeType.ANDROID_ASSETS, settings);
+    }
+
+    public GVRSceneObject loadJassimpModelFromURL(String urlString) throws IOException {
```

Comment:
(2)
And then, if you happen to have something like what I suggested in comment (1), users should be able to use one version of loadJassimpModel which accepts a GVRAndroidResource. And on a side note, I think we should rename the method to loadModel as the loader we use is irrelevant.